modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/utils/MinMuleVersionUtils.java

```diff
@@ -535,6 +560,18 @@ private static ResolvedMinMuleVersion calculateParameterTypeMinMuleVersion(Type
     return parameterMMV;
   }
 
+  private static boolean isLinkageError(Throwable t) {
+    t = t.getCause();
+    while (t.getCause() != null) {
```

Comment:
bad condition. Some times errors end up having themselves as a their cause. If you're going to do this you need a set of seen causes already

Reply:
Done.